bundles/org.openhab.core/src/test/java/org/openhab/core/library/types/HSBTypeTest.java

```diff
@@ -129,14 +129,8 @@ public void testConversionToPointType() {
     public void testConversionToXY() {
         HSBType hsb = new HSBType("220,90,50");
         PercentType[] xy = hsb.toXY();
-        assertEquals(new PercentType("16.969364"), xy[0]);
-        assertEquals(new PercentType("12.379659"), xy[1]);
-    }
-
-    @Test
-    public void testCreateFromXY() {
```

Comment:
I have removed this test because it was invalid. The JavaDoc of `HSBType.fromXY` states that the parameters should be between 0 and 1.